### PR TITLE
Using cryptography lib instead of (dead) pycrypto

### PR DESCRIPTION
**Pull request by tuxuser**
*Added unittest for laf crypto
*Made the import error more descriptive, incase laf_crypto fails to import.
*Switch to cryptography lib

### CODE REVIEW

**Comment by Lekensteyn** [comment]
Changes look sane, but is this tested on Windows as well? `pip install cryptography` *should* work in theory too when using the official Python distribution right? (Maybe this is worth including in the error message?)

**Comment by gjdunga** [comment]
I will pull and see... I have to clean my system up. Every so often, I have to wipe it. I am way too messy...

**Comment by tuxuser** [comment]
Late reply: tested flawless installation of cryptography Lib on windows X86 yesterday!

**Comment by Lekensteyn** [comment]
Would it also be possible to update the installation instructions in the README?

**Comment by tuxuser** [comment]
Done ;)
bonus feature: py2 compatibility :P

File: laf_crypto.py [hunk 1]
```diff
@@ -6,19 +7,21 @@
 def key_transform(old_key):
     new_key = b''
     for x in range(32, 0, -1):
-        new_key += int_as_byte(old_key[x-1] - (x % 0x0C))
+        c = old_key[x-1]
+        if not isinstance(c, int):
+            c = ord(c)
```
Comment (Lekensteyn) [inline laf_crypto.py]: fyi, if you use `old_key = bytearray(old_key)` outside the loop, then indexing `old_key` will give you integers on both Python 2 and 3

**Comment by Lekensteyn** [comment]
Yes, looks fine now, thanks!

**Pull request merged**